composer.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "appwrite/playground-for-php",
+    "authors": [
+        {
+            "name": "sparkingdark",
+            "email": "debomastet335@gmail.com"
+        }
+    ],
+    "config": {
+        "platform": {
+            "php":"^5.6.1"
```

Comment:
That's not, how requiring a PHP Version works. Please remove the whole config block and add PHP to the require block as follows:

```
"php": "^5.6"
```

Reply:
done